saleor/order/utils.py

```diff
@@ -565,21 +546,21 @@ def change_order_line_quantity(
         line.quantity = new_quantity
         total_price_net_amount = line.quantity * line.unit_price_net_amount
         total_price_gross_amount = line.quantity * line.unit_price_gross_amount
-        line.total_price_net_amount = total_price_net_amount.quantize(Decimal("0.001"))
-        line.total_price_gross_amount = total_price_gross_amount.quantize(
-            Decimal("0.001")
+        line.total_price_net_amount = quantize_price(total_price_net_amount, currency)
```

Comment:
Hhm, do we actually need to make these calculations here? 
I mean, we agree that the base price is static, but we still need to get tax values from TaxPlugin/TaxApp/FlatRates. So when we change the `order_line_quantity`, we change the `_price` fields, then later we set `price_expiration` to True, and we calculate prices in the resolver. Right now I am thinking: do we get something here? The prices are marked as expired, so they will be calculated once again, right?

Reply:
Correct. IMO this should be handled in another PR. There is already an internal task for this: https://linear.app/saleor/issue/EXT-1729/all-order-mutation-should-be-lazy. 
I don't want to extend this PR further as it already introduces a massive change. What do you think?

Reply:
IMO separate task sounds good